### DISCUSSION TITLE
Extend model:show artisan command information

### DISCUSSION THREAD

**Discussion by dorkster100** [laravel/framework#46957] [Ideas]
It would be nice to have more information displayed in the model:show artisan command, especially things that are "auto-discovered". For example Policies, perhaps Migrations, Observers? There definitely are more things to display, many of which may not be easily read from the code, that are "convention" based.

This could be perhaps also used by IDE plugins etc, to make it easier to navigate code.

**Comment by dorkster100** [comment]
All of the things mentioned except migrations are already implemented.

Policy has a bug that I submitted PR for.

**Reply by bjuppa** [reply]
I'd like `artisan model:show` to list global scopes, and perhaps also available scopes for the model.

I couldn't find any discussions about this in this repo, does anyone know if it has been considered?
